src/components/ha-dialog-footer.ts

```diff
@@ -0,0 +1,34 @@
+import { css, html, LitElement } from "lit";
+import { customElement } from "lit/decorators";
+
+@customElement("ha-dialog-footer")
+export class HaDialogFooter extends LitElement {
+  protected render() {
+    return html`
+      <footer class="footer">
+        <slot name="secondaryAction"></slot>
+        <slot name="primaryAction"></slot>
+      </footer>
```

Comment:
What would be interesting for me here is to have some kind of default styling when we use buttons. 

The primary action is always `accent` and the secondary are `plain`. It's just an idea and I am not sure how the API should look like. But we could force a bit how dialog actions should look like. 

Moreover it would be interesting to think of if the footer should listen for enter or automatically should focus the primary action, except it's defined differently.

Reply:
At one point for this I did setup a primary and secondary action object configs (which had these defaults) but had issues with the click action passing back to the parent, so decided to simplify to this.

The dialog does listen for the enter key when ha-dialog-footer is defined with a primary slot, idea here is to make it so the footer is used in conjunction (when needed) with the new dialog

Reply:
Added note to jsdoc for this